client/shipping-label/views/dialog.js

```diff
@@ -1,14 +1,73 @@
 import React from 'react';
 import Dialog from 'components/dialog';
+import ActionButtons from 'components/action-buttons';
+import { translate as __ } from 'lib/mixins/i18n';
+import sum from 'lodash/sum';
+import isArray from 'lodash/isArray';
+import isPlainObject from 'lodash/isPlainObject';
+import some from 'lodash/some';
+import values from 'lodash/values';
+import AddressStep from './steps/address';
+import PackagesStep from './steps/packages';
+import RatesStep from './steps/rates';
+import PreviewStep from './steps/preview';
+
+const hasErrors = ( tree ) => {
+	if ( ! tree ) {
+		return false;
+	}
+	if ( isArray( tree ) ) {
+		return some( tree, hasErrors );
+	}
+	if ( isPlainObject( tree ) ) {
+		return some( values( tree ), hasErrors );
+	}
+	return true;
+};
 
 const PrintLabelDialog = ( props ) => {
+	const currencySymbol = props.storeOptions.currency_symbol;
+
+	const getTotalCost = () => {
+		const ratesInfo = props.form.rates;
+		const ratesCost = ratesInfo.values.map( ( rateId, index ) => ratesInfo.available[ index ][ rateId ].rate );
+		return sum( ratesCost ).toFixed( 2 );
+	};
+
+	const canPurchase = ! props.form.isSubmitting && ! hasErrors( props.errors );
 	return (
 		<Dialog
 			isVisible={ props.showDialog }
 			onClose={ props.labelActions.exitPrintingFlow }
-			additionalClassNames="wcc-modal wcc-shipping-label-dialog"
-		>
-			<span>Not implemented yet</span>
+			additionalClassNames="wcc-modal wcc-shipping-label-dialog" >
+			<AddressStep.Origin
```

Comment:
Some markup is missing.  As part of this PR, let's include the markup for
* header, body and footer containers
* the dialog title "Create shipping labels" in the header container
* the left and right hand containers/"columns" (preview goes in the right hand one and all of the other steps inside the left) inside the body container
* the action buttons inside the footer container

Or something like that. We can fine tune styles later, but the markup defining those containers should be in this PR imho

Please also increase the dialog width by 50%